funread_backend/Badges/urls.py

```diff
@@ -0,0 +1,14 @@
+from Badges import views
+from django.urls import path
+from rest_framework.urlpatterns import format_suffix_patterns
+
+urlpatterns = [
+    path('api/badges/create/', views.create_badge),
+    path('api/badges/listar/', views.list_badges),
+    path('api/badges/update/<int:badge_id>/', views.update_badge),
+    path('api/badges/delete/<int:badge_id>/', views.delete_badge),
+    path('api/badges/list_user_badges_with_status/<int:user_id>/', views.list_user_badges_with_status)
```

Comment:
Estas no son las mejores prácticas para crear endpoints (ver https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5). No es necesarios agregar sufijos como crear, listar, actualizar, eliminar, el método que le asignamos al endpoint es el que nos va a decir la acción que se va a tomar.

El último ejemplo en la corrección hay que revisarlo.


```suggestion
urlpatterns = [
    path('api/badges/', views.create_badge),
    path('api/badges/', views.list_badges),
    path('api/badges/<int:badge_id>/', views.update_badge),
    path('api/badges/<int:badge_id>/', views.delete_badge),
    path('api/badges/user/<int:user_id>/', views.list_user_badges_with_status)
```